File: oss/src/main/java/com/truenine/component/oss/properties/AliCloudOssProperties.java
package com.truenine.component.oss.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Data
@ConfigurationProperties(prefix = "component.oss.ali-cloud")
public class AliCloudOssProperties {
  String accessKey;
  String endpoint;
  String accessKeySecret;
  String bucketName;
}

File: oss/src/main/kotlin/com/truenine/component/oss/properties/AliCloudOssProperties.kt
package com.truenine.component.oss.properties

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "component.oss.ali-cloud")
data class AliCloudOssProperties(
  var accessKey: String?,
  var endpoint: String?,
  var accessKeySecret: String?,
  var bucketName: String?
) {

}
